api/envoy/config/core/v3/base.proto

```diff
@@ -346,9 +346,9 @@ message HeaderValueOption {
 
   // Describes the supported actions types for header append action.
   enum HeaderAppendAction {
-    // This action will append the specified value to the existing values if the header
-    // already exists. If the header doesn't exist then this will add the header with
-    // specified key and value.
+    // For headers predefined inline, this action will append the specified value to the
```

Comment:
thanks! and what about this:

```
This action will append the specified value to the existing values if the header predefined inline already exists. If the header doesn't exist then this will add the header with specified key and value.
```
@tyxia

Reply:
How about below ?
```
If the header already exists, this action will result in: 
  - Comma concatenated for predefined inline headers.
  - Duplicate header added in the HeaderMap for other headers.
 
If the header doesn't exist, this action will add new header with specified key and value.
```

cc @wbpcode

Reply:
SGTM.

Reply:
done, thanks. @tyxia @wbpcode